README.md

```diff
@@ -1,20 +1,54 @@
-# WaffleHacks Landing Page
+<p align="center">
+  <a href="https://www.gatsbyjs.com/?utm_source=starter&utm_medium=readme&utm_campaign=minimal-starter">
+    <img alt="Gatsby" src="https://www.gatsbyjs.com/Gatsby-Monogram.svg" width="60" />
+  </a>
+</p>
+<h1 align="center">
+  Gatsby minimal starter
+</h1>
 
-This is the landing page for the Waffle Hacks hackathon.
-It is a modified version of the [Gatsby Tailwind Starter](https://github.com/app-generator/gatsbyjs-starter-tailwindplay) made by AppSeed.
+## 🚀 Quick start
 
-The countdown clock is a modified version of the countdown timer in [this blog post](https://www.florin-pop.com/blog/2019/05/countdown-built-with-react/).
```

Comment:
Rather than replacing the old readme with the autogenerated one from Gatsby, it would be better to just update the description of the old one saying that this is a temporary version.